### ISSUE TITLE
使用2.0rc训练cifar10出错

### ISSUE THREAD

**Issue by JimXu1989** [PaddlePaddle/Paddle#30534]
仿照官方文档训练MNIST的例程，我使用model.fit()函数来训练cifar10,程序如下，却无法正常训练，总是的到下面的结果：
The loss value printed in the log is the current step, and the metric is the average value of previous step.
Epoch 1/2
step 1563/1563 [==============================] - loss: 0.0000e+00 - acc: 0.1000 - 12ms/step          
Epoch 2/2
step 1563/1563 [==============================] - loss: 0.0000e+00 - acc: 0.1000 - 12ms/step 

代码如下，其中前半部分与官方例程一样，后面是自己移花接木
import paddle
import paddle.nn.functional as F
from paddle.vision.transforms import ToTensor
import numpy as np
import matplotlib.pyplot as plt

print(paddle.__version__)

transform = ToTensor()
cifar10_train = paddle.vision.datasets.Cifar10(mode='train',
                                               transform=transform)
cifar10_test = paddle.vision.datasets.Cifar10(mode='test',
                                              transform=transform)

class MyNet(paddle.nn.Layer):
    def __init__(self, num_classes=1):
        super(MyNet, self).__init__()

        self.conv1 = paddle.nn.Conv2D(in_channels=3, out_channels=32, kernel_size=(3, 3))
        self.pool1 = paddle.nn.MaxPool2D(kernel_size=2, stride=2)

        self.conv2 = paddle.nn.Conv2D(in_channels=32, out_channels=64, kernel_size=(3,3))
        self.pool2 = paddle.nn.MaxPool2D(kernel_size=2, stride=2)

        self.conv3 = paddle.nn.Conv2D(in_channels=64, out_channels=64, kernel_size=(3,3))

        self.flatten = paddle.nn.Flatten()

        self.linear1 = paddle.nn.Linear(in_features=1024, out_features=64)
        self.linear2 = paddle.nn.Linear(in_features=64, out_features=num_classes)

    def forward(self, x):
        x = self.conv1(x)
        x = F.relu(x)
        x = self.pool1(x)

        x = self.conv2(x)
        x = F.relu(x)
        x = self.pool2(x)

        x = self.conv3(x)
        x = F.relu(x)

        x = self.flatten(x)
        x = self.linear1(x)
        x = F.relu(x)
        x = self.linear2(x)
        return x

epoch_num = 10
batch_size = 32
learning_rate = 0.001

from paddle.metric import Accuracy
model = paddle.Model(MyNet())   # 用Model封装模型
optim = paddle.optimizer.Adam(learning_rate=0.001, parameters=model.parameters())

# 配置模型
model.prepare(
    optim,
    paddle.nn.CrossEntropyLoss(),
    Accuracy()
    )

# 训练模型

model.fit(cifar10_train,
        epochs=2,
        batch_size=32,
        verbose=1
        )

看起来没有什么问题，可是就是训练不成功
